### ISSUE TITLE
Add missing services to reduce (down to 0) the skipped tests

### ISSUE THREAD

**Issue by stronk7** [moodlehq/moodle-ci-runner#15]
This issue is about to reduce the number of skipped tests with phpunit. Namely:

- Solr (#12). Was commented out because there were some problems with our php images, now solved https://github.com/moodlehq/moodle-php-apache/pull/58
- Memcached (#13).
- MongoDB (#14).
- mlbackend-python. Now that the python backend can run via server ([MDL-66004](https://tracker.moodle.org/browse/MDL-66004))... we can add it to the runner. Worth discussing about adopting and automating the managing of that image with @dmonllao .

Basically, that's all we need to get all tests passing. Let's use this issue as epic to control it.

Ciao :-)

**Comment by stronk7** [comment]
Have just added 4 PRs that should be merged in order:

#11 - tiny change to better detect the script directory. Same approach we use in a lot of local_ci scripts, enable you to wrap it in other, caller scripts...
#12 - enable solr.
#13 - enable memcached.
#14 - enable mongodb

Ciao :-)

**Comment by andrewnicols** [comment]
All looking good and merged \o/
Win! Nice work Eloy.

**Comment by stronk7** [comment]
Thanks for merging, @andrewnicols !

If you don't mind, I'm going to keep this open to take a look to the pending stuff:

- mlbackend-python. Now that the python backend can run via server (MDL-66004)... we can add it to the runner. Worth discussing about adopting and automating the managing of that image with @dmonllao .
- uuid
- unoconv

And linking this with https://github.com/moodlehq/moodle-docker/issues/110 that is aiming for exactly the same, but from moodle-docker.

**Comment by stronk7** [comment]
With the 6 PRs above all our (non-EOL) images should get uuid tests passing.

So, remaining:

- mlbackend-python
- unoconv

Ciao :-)

**Comment by stronk7** [comment]
With #73 merged, now the only remaining tests are:
* unoconv

Ciao :-)

**Comment by stronk7** [comment]
Uhm... was revisiting this to see if finally can get it closed... and I've just seen that phpunit is still skipping the mlbackend_python tests:

https://ci.moodle.org/view/B%20-%20master/job/W.01.01%20-%20PHPUnit%20-%20Postgres/2183/console

I thought we were running them... looking...
